book/src/csi-driver-object.md

```diff
@@ -52,8 +52,13 @@ To install, a CSI driver's deployment manifest must contain a `CSIDriver`
 object as shown in the example above.
 
 >NOTE: The cluster-driver-registrar side-car which was used to create CSIDriver
->objects in Kubernetes 1.13 is being redesigned for Kubernetes 1.15 and
->therefore not available for Kubernetes 1.14.
+>objects in Kubernetes 1.13 has been deprecated for Kubernetes 1.16. No
+>cluster-driver-registrar has been released since Kubernetes 1.14.
+
+`CSIDriver` instance should exist for whole lifetime of all pods that use
+volumes provided by corresponding CSI driver, so Kubernetes knows what
+`volume_context` to pass to the pods and that `Controller(Un)PublishVolume`
```

Comment:
Also do those pages need update?

Reply:
> Also do those pages need update?

I think the're OK. The both have this:

> CSIDriver object should be manually included in the driver deployment manifests.

> Previously, the cluster-driver-registrar sidecar container could be deployed to automatically create the object. Once the flags to this container are configured correctly, it will automatically create a CSIDriver Object when it starts with the correct fields set.

https://kubernetes-csi.github.io/docs/skip-attach.html#skip-attach-with-csi-driver-object says

Reply:
> Point to pod info and skip attach feature pages instead?

Replaced with links

￼
￼￼